### DISCUSSION TITLE
max_bytes_length_exceeded_exception for query packs in FleetDM in Security Onion

### DISCUSSION THREAD

**Discussion by sheetalmilind** [Security-Onion-Solutions/securityonion#8832] [Unsupported Versions]
We are able to see the max_bytes_length_exceeded_exception in logstash logs for the default query pack - windows-pack available inside FleetDM. This issue is mostly coming for the snapshot queries where the data is large. Can you please let us know is there any solution for this.

**Comment by defensivedepth** [comment]
Does the `max_bytes_length_exceeded_exception` have a specific field that it lists?

**Comment by sheetalmilind** [comment]
Hi
 Yes, this issue occurs for message field.

Regards,
Sheetal

**Reply by defensivedepth** [reply]
I would suggest trying to narrow down which query is generating large amounts of data and see if there is a way to tweak / tune it. The default osquery queries, (just like any default rules etc), need to be tweaked / tuned for your specific environment.
